components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/Config.kt

```diff
@@ -15,15 +19,19 @@ import org.mozilla.fxaclient.internal.Config as InternalConfig
 class Config internal constructor(internal val inner: InternalConfig) : AutoCloseable {
 
     override fun close() {
+        job.cancel()
         inner.close()
```

Comment:
With the scope as part of the companion object wouldn't this cancel **all** Config related jobs? Not sure, maybe we want this?

Reply:
Yes, it will. This seems OK as we're only having one short-lived Config instance per App. My concern was that `cancel` might prevent future submissions, but it worked when testing.

Reply:
Since an instance-based scope wouldn't make sense here, an alternative to this solution would be to use `GlobalScope.async`, but then technically coroutines could be around longer. This is all just for a one-time method call though, so it seems fine either way?

Reply:
Actually, I am wrong about `job.cancel`. Once cancelled, it's over. I will push an update for this.

Reply:
OK, updated, since this is a single static method call to create the singleton config, I think `GlobalScope` is fine in this case. In the future, either the higher-level API needs to provide a single managed scope, or we push this to the consumer and make all calls synchronous. Either way, we don't want the app having to coordinate multiple scopes.